docs/reference/access-control-lists.md

```diff
@@ -1,37 +1,52 @@
-### The new ACL model
-As mentioned in our [Security Update](https://docs.lakefs.io/posts/security_update.html#whats-changing) the coming lakeFS version introduces a simplified authorization mechanism based on ACLs.
+---
+layout: default
+title: Access Control Lists (ACL)
+parent: Reference
+description: Access Control Lists (ACL)
+nav_order: 100
+has_children: false
+---
 
-### Permissions
-The new ACL model defines permissions at the group level, 
-we'll review the new permissions compared to their previous representation:
+## Access Control Lists (ACL) in lakeFS
+
+ACLs were introduced in their current form in v0.97 of lakeFS as part of [changes to the security model](/posts/security_update.html#whats-changing) in lakeFS. 
+
+## ACL
+
+You can attach Permissions and scope them to groups in the Groups page.
+There are 4 default groups, named after the 4 permissions. Each group is global (applies for all repositories).
+
+| Group ID    | Allows | Repositories |
+|-------------|--------------------|--------------|
+| **Read**    | Read operations, creating access keys               | All          |
+| **Write**   | Allows all data read and write operations.              | All          |
+| **Super**   | Allows all operations except auth.              | All          |
+| **Admin**   | Allows all operations.              | All          |
```

Comment:
This is great, and I agree that it will be more useful to new users.  However _existing_ users should know that group names have changed.  For instance, they might have placed a group name in their configuration as a "default" group.

Reply:
Isn't that covered by [the table](https://github.com/treeverse/lakeFS/pull/5443/files#diff-b4af88dc5e8cf5c3c51d9d2a9871772227453b332565adfdca13594d3647e20cR38) later in the doc?